docs/resources/server/advancements.md

```diff
@@ -247,7 +247,7 @@ builder.requirements(AdvancementRequirements.allOf(List.of("pickup_dirt")));
 
 // Save the advancement to disk, using the given resource location. This returns an AdvancementHolder,
 // which may be stored in a variable and used as a parent by other advancement builders.
-builder.save(saver, ResourceLocation.fromNamespaceAndPath("examplemod", "example_advancement"), existingFileHelper);
+builder.save(consumer, ResourceLocation.fromNamespaceAndPath("examplemod", "example_advancement"), existingFileHelper);
```

Comment:
I disagree with this change. `AdvancementGenerator#generator` passes a `Consumer<AdvancementHolder> saver`, which is the name of the parameter in NeoForge. I don't like consumer in general because it doesn't explain what the lambda is supposed to be doing.